js/hview.js

```diff
@@ -1046,30 +1046,22 @@ $(function () {
                 icon: showHiddenNodes ? "ic_checked" : "ic_unchecked",
                 id: 0
             },
+            null,
             {
                 text: "Dark theme",
                 icon: isDarkTheme() ? "ic_checked" : "ic_unchecked",
                 id: 5
-            },
-            null,
-            {
-                text: "Save hierarchy",
-                icon: "ic_save",
-                id: 1
-            },
-            {
-                text: "Refresh",
-                icon: "ic_refresh",
-                id: 2
             }
         ];
+
         if (!$("#hviewtabs").is(":visible")) {
             // Only show the preview menu when tabs are not available
             menu.unshift({
                 text: "Preview",
                 icon: "ic_submenu",
                 id: 6
-            });
+            },
+            null);
```

Comment:
probably not at the divider at the end otherwise you would get two dividers if this is the last entry

Reply:
In this particular case, the divider goes afterwards because we are prepending the menu item, rather than appending it here.